lighthouse-core/test/index-test.js

```diff
@@ -37,6 +37,147 @@ describe('Module Tests', function() {
     assert.notEqual(lighthouseTraceCategories.length, 0);
   });
 
+  describe('lighthouse', () => {
+    it('should throw an error when the first parameter is not defined', function() {
```

Comment:
can you modify these tests (or at least some of them) to use `expect(() => lighthouse(...)).toThrow(...)`?